scripts/o3de/ExportScripts/export_utility.py

```diff
@@ -0,0 +1,195 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+
+import o3de.command_utils as command_utils
+import o3de.export_project as exp
+import o3de.manifest as manifest
+
+from o3de import android, android_support
+
+
+import argparse
+import logging
+import pathlib
+import sys
+
+from typing import List
+
+def process_level_names(ctx, seedfile_paths, level_names, platform):
+    for level in level_names:
+        seedfile_set = set(seedfile_paths)
```

Comment:
`seedfile_set` should be outside of this loop. Otherwise its inefficiently creating a new set for the `seedfile_paths` at every iteration of the loop